src/entities/addLiquidity/types.ts

```diff
@@ -64,8 +64,12 @@ export type AddLiquidityBaseBuildCallInput = {
     wethIsEth?: boolean;
 } & AddLiquidityBaseQueryOutput;
 
+export type AddLiquidityV3BuildCallInput = AddLiquidityBaseBuildCallInput & {
+    userData: Hex;
```

Comment:
I am curious why not just make userdata optional here instead of throwing errors for if it is missing?

```sugggestion
userData?: Hex;

Reply:
That's what I suggested on slack - for now I think I'll keep them required so I don't affect overall behavior that much - we can make it optional on a following PR.